TOMP-API.yaml

```diff
@@ -187,7 +187,8 @@ paths:
             Expires:
               description: The result is valid until this timestamp. The pending booking is expired after this timestamp. This option can be used, but there is also a facility to use the webhook, mentioned in '#/component/schemas/booking'.
               schema:
-                $ref: "#/components/schemas/timestamp"
+                type: string
+                format: http-date
               required: true
```

Comment:
what's http-date?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expires in description